.infra/eclipse/junit-eclipse-formatter-settings.xml

```diff
@@ -161,7 +162,7 @@
 <setting id="org.eclipse.jdt.core.formatter.insert_space_before_colon_in_for" value="insert"/>
 <setting id="org.eclipse.jdt.core.formatter.alignment_for_resources_in_try" value="80"/>
 <setting id="org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations" value="false"/>
-<setting id="org.eclipse.jdt.core.formatter.alignment_for_selector_in_method_invocation" value="0"/>
+<setting id="org.eclipse.jdt.core.formatter.alignment_for_selector_in_method_invocation" value="48"/>
```

Comment:
@nicolaiparlog What does this do exactly?

Reply:
See c8a82b2:

> * make `//@Formatter:off` before each stream pipeline unnecessary
>   ~> in Eclipse's formatter configuration UI, this is the corresponding setting:
>      Line Wrapping > Wrapping Settings > Function Calls > Qualified Invocations
>   (there are two magic values - 80 is ok as well)
> 
>        <setting id="org.eclipse.jdt.core.formatter.alignment_for_selector_in_method_invocation" value="48"/>

We fiddled with the settings for a while before we settled on this one.